### DISCUSSION TITLE
Alpine Expression Error: wireui_dropdown is not defined

### DISCUSSION THREAD

**Discussion by souzadl** [wireui/wireui#666] [BUG]
![image](https://github.com/wireui/wireui/assets/33837410/5027e38c-cc91-478d-bfa8-93614290ab48)

Estou iniciando um projeto laravel com breeze + wireui.
Por um breve período de tempo funciona certinho, porém passa a apresentar esse erro.
Já tentei algumas coisa mas não consegui entender o problema nem achar uma solução.

**Comment by keepanitreel** [comment]
Laravel Breeze component names are conflicting with wireui

in config/wireui.php prefix the components alias with something like wireui

Its a known issue #626
